CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_google_play_source v0.1.1
+
+## Fixes
+[PR #8](https://github.com/fivetran/dbt_google_play_source/pull/8) incorporates the following change:
+- The LinkedIn Ads schema and database variables were incorrectly documented within the README. The README has been updated to reflect the correct variable names.
+
```

Comment:
@johnf I think this may be a copy/paste holdover from the latest Ad Reporting CHANGELOG entry. Would you be able to update the entry to reflect the changes within this PR?

Reply:
It was, I copied from there to get the right format :)
Fixed